src/add-remove.js

```diff
@@ -90,6 +90,7 @@ export const addTask = (description, index, check) => {
   list.addEventListener('click', (event) => {
     editTask(event.target.id, list, checkboxImage, checkboxLabel);
   });
+  // console.log(list);
```

Comment:
- [ ] @team kindly remove the commented code, it's not good practice to have commented code in your codebase.